psga/common/get_pangolin_data_dir.nf

```diff
@@ -0,0 +1,22 @@
+/*
+ * Obtain the full path to the pangolin-data directory from the Reference Data CSV
+ * file.
+ * pangolin-data is distributed as a Python package and pangolin will use it by
+ * default when the data directory is not specified in the call (i.e. no "--datadir").
+ * We want to control the version of pangolin-data independently of the version of
+ * the pipeline, so we pass it as Reference Data.
+ */
+process get_pangolin_data_dir {
```

Comment:
Every process spins up it's own docker container to run, hence we've been inlining them. However, if you set the executor on this process to local I think it gets around that.